From 4cbea8c6de4a322ad54d895572dbf883f00a84ff Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Fri, 30 Oct 2015 17:25:28 +0800
Subject: [PATCH 0651/5242] MLK-11802-3 ARM: imx6ul-14x14-evk: Enable dual
 switch function for USB

commit  0eeb6249cb350b762144bcccde69e6f4c4ab142e from
https://source.codeaurora.org/external/imx/linux-imx.git

Since we have supported control vbus through ID pin, we can
support dual switch function for otg port, but fully otg can't
be supported at this port.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 743da29..17f0da2 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -368,6 +368,9 @@
 
 &usbotg1 {
 	dr_mode = "otg";
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

